.github/workflows/sync-main-to-preprod-and-prod.yml

```diff
@@ -64,32 +66,18 @@ jobs:
         run: |
           gcloud auth login --brief --cred-file="${{ steps.auth.outputs.credentials_file_path }}"
           gcloud auth list
-      # Sync
-      - name: sync
+      - name: diff-main-prod
```

Comment:
Name is for display purposes, so something like "Check update files" or something?